src/Opdex.Platform.Infrastructure/Data/Handlers/MiningGovernances/Nominations/SelectActiveGovernanceNominationsByIdQueryHandler.cs

```diff
@@ -27,21 +27,21 @@ public class SelectActiveGovernanceNominationsByGovernanceIdQueryHandler
                 {nameof(MiningGovernanceNominationEntity.ModifiedBlock)}
             FROM governance_nomination
             WHERE {nameof(MiningGovernanceNominationEntity.IsNominated)} = true AND
-                  {nameof(MiningGovernanceNominationEntity.GovernanceId)} = @{nameof(SqlParams.GovernanceId)}
-            LIMIT {GovernanceConstants.MaxNominations};";
+                  {nameof(MiningGovernanceNominationEntity.MiningGovernanceId)} = @{nameof(SqlParams.MiningGovernanceId)}
+            LIMIT {MiningGovernanceConstants.MaxNominations};";
 
         private readonly IDbContext _context;
         private readonly IMapper _mapper;
 
-        public SelectActiveGovernanceNominationsByGovernanceIdQueryHandler(IDbContext context, IMapper mapper)
+        public SelectActiveMiningGovernanceNominationsByIdQueryHandler(IDbContext context, IMapper mapper)
```

Comment:
This name and the associated query should be `ByMiningGovernanceId`. Might need this change for Retrieve etc. too

Reply:
Good point, nominations have an id so `ById` isn't clear